extensions/wikia/FounderEmails/events/FounderEmailsEditEvent.class.php

```diff
@@ -43,151 +147,121 @@ public function enabled ( $wgCityId, User $user ) {
 		}
 
 		// If digest mode is enabled, do not create edit event notifications
-		if ( $user->getOption( "founderemails-complete-digest-$wgCityId" ) ) {
+		if ( $user->getOption( "founderemails-complete-digest-$wikiId" ) ) {
 			return false;
 		}
-		if ( $user->getOption( "founderemails-edits-$wgCityId" ) ) {
+
+		if ( $user->getOption( "founderemails-edits-$wikiId" ) ) {
 			return true;
 		}
+
 		return false;
 	}
 
```

Comment:
As seen above, the ```process``` method was broken up into smaller methods.  More can be removed once we finish the final email sent via this code (https://wikia-inc.atlassian.net/browse/SOC-767)